### DISCUSSION TITLE
Actions hanging on Starting your workflow runs...

### DISCUSSION THREAD

**Discussion by roni-frantchi** [community/community#26956] [Actions]
<p>Anyone else experience it?.. our whole CD pipeline is stuck :\</p>

**Comment by anthonator** [comment]
<p>We’re also seeing this</p>

**Comment by wipe2238** [comment]
<blockquote>
<p><a class="mention" href="/roni-frantchi">@roni-frantchi</a> wrote:</p>
</blockquote>
<blockquote>
<p>Anyone else experience it?</p>
</blockquote>
<p>Yup. 10 minutes of waiting before workflow starts.</p>

**Comment by davidjdixon** [comment]
<p>I’m also seeing this across several different workflows being triggered by different users in our account. Has been like this for at least a few hours.</p>
<p>Anyone from Github staff able to give an idea on whats going on (GitHub status not reporting anything out of the ordinary)?</p>

**Comment by roni-frantchi** [comment]
<p>Our actually starts after a few hours, or never…</p>

**Comment by StefMa** [comment]
<p>Yep. Same here… :frowning:</p>

**Comment by grochockik** [comment]
<p>same here and still ongoing</p>

**Comment by tomschlick** [comment]
<p>Seeing this as well :frowning:</p>

**Comment by runspired** [comment]
<p>Same. somewhere between 20min and 4 hours before it’ll actually start. The dangerous part is because it’s queued but not started PRs don’t show the checks and thus appear passing when no tests have been run.</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hi,</p>
<p>Very sorry for the trouble! We have an issue open internally and our engineers are working on it, we will update you again as soon as we have more information to share.</p>

**Comment by zellski** [comment]
<p>For what it’s worth – I’m getting this now again. It worked fine just a few hours ago.</p>
<p><a class="mention" href="/andreagriffiths11">@andreagriffiths11</a> did you ever resolve the internal issue, or is it sporadic?</p>
<p>Zell</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Thanks for the ping <a class="mention" href="/zellski">@zellski</a>, this issue should be resolved. If you are still having issues please contact us at <a href="https://github.com/contact" rel="nofollow noopener">https://github.com/contact</a> and we’ll take a look.</p>

**Comment by zellski** [comment]
<p>Thanks Andrea – yeah, turns out I’d just screwed up the contents of “runs-on” line, which fails the workflow without feedback:</p>
<p><img src="https://user-images.githubusercontent.com/714167/181100959-4743f112-a0d6-47a1-b732-1d26b1cc51a4.png" alt="Screenshot 2019-11-05 11.24.45.png" data-base62-sha1="bLEvVL8xmvMlN5fMvFBqPN1GhUL" width="372" height="104"></p>

**Comment by artivilla** [comment]
<p>Any update on this?</p>

**Comment by BaronChiao** [comment]
<p>Excuse me, is this question solved?</p>
<p>same here.</p>

**Comment by koppor** [comment]
<p>Also experiencing this. Added a GitHub action on a branch. It is not available on the master branch (yet). Wanting to start with a pull request.</p>
<p>Solution: It was an invalid workflow file. Just had to wait a bit, refresh the page and then the error was shown.</p>
